azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md

```diff
@@ -60,6 +60,8 @@ Only one Arc Resource Bridge can be deployed on a cluster. Each Azure Stack HCI
 Deploying Azure Arc Resource Bridge requires the following:
 
 - The latest version of Azure CLI installed on all servers of the cluster.
+  - To install Azure CLI on each cluster node use RDP connection.
+  - Follow the instructions on this KB: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-windows
```

Comment:
This is an article, not a KB, and the link should be relative, i.e. /cli/azure/install-azure-cli-windows

Reply:
Done, replace KB by article and relative URI.